openstack/examples/os-create.yml

```diff
@@ -5,15 +5,18 @@
 
   vars:
 
-    omero_vm_name: test-idr-omero
-    omero_vm_key_name: SSH keypair name
     omero_vm_image: CentOS 7.2 1602
-    omero_vm_flavour: m1.medium
-
-    setup_script: idr-example-omero-bootstrap.sh
+    omero_vm_flavour: m1.large
+    omero_vm_groups: "centos,omero"
```

Comment:
Thoughts on making it clearer these are groups, e.g. `centos-vms`, `omero-instances`, etc, or something else?